### PR TITLE
Renamed correct titles for test methods.

### PR DESCRIPTION
**Pull request by SofiiaKazanivska**
Renamed correct titles for test methods.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>mate-academy</groupId>
-    <artifactId>registration-validation</artifactId>
+    <artifactId>diamond</artifactId>
```
Comment (aaananas) [inline pom.xml:8]: you should not change pom

File: src/main/java/core/basesyntax/service/RegistrationServiceImpl.java [hunk 2]
```diff
@@ -5,10 +5,32 @@
...
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() < MIN_AGE) {
+            throw new RuntimeException("The user must be older than " + MIN_AGE);
+        }
+        if (user.getPassword().length() < MIN_PASSWORD_LENGTH) {
+            throw new RuntimeException("The password length must be longer than "
+                    + MIN_PASSWORD_LENGTH);
+        }
```
Comment (aaananas) [inline src/main/java/core/basesyntax/service/RegistrationServiceImpl.java:20]: what if password is null 
there will be NPE
check if the password is null before checking its length

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
```
Comment (aaananas) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:14]:
```suggestion
    private RegistrationService registrationService;
```

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    void register_userName_Ok() {
+        User actual1 = registrationService.register(alice);
+        assertTrue(Storage.people.contains(actual1));
+        User actual2 = registrationService.register(bob);
+        assertTrue(Storage.people.contains(actual2));
+    }
+
+    @AfterEach
+    void register_cleanStorage()
+    {
```
Comment (aaananas) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:82]:
```suggestion
    void cleanStorage() {
```

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        bob.setPassword("123Bob");
+        bob.setAge(22);
+    }
+
+    @Test
+    void register_ageLessThan18_notOk() {
+        alice.setAge(15);
+        assertThrows(RuntimeException.class, () -> {
+            registrationService.register(alice);
+        });
```
Comment (aaananas) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:39]:
```suggestion
        assertThrows(RuntimeException.class, () -> 
            registrationService.register(alice));
```
there is just 1 operation
fix other methods accordingly

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        alice.setPassword("123Alice");
+        alice.setAge(18);
+
+        bob.setLogin("Bob");
+        bob.setPassword("123Bob");
+        bob.setAge(22);
+    }
+
+    @Test
+    void register_ageLessThan18_notOk() {
```
Comment (aaananas) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:35]: dont put concrete values to the test method name
the minimal age can be changed in the future

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+        });
+        bob.setAge(-1);
+        assertThrows(RuntimeException.class,
+                () -> {
+                    registrationService.register(bob);
+                });
+    }
+
+    @Test
+    void register_passwordLeast6Characters_notOk() {
```
Comment (aaananas) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:48]: the same as with the test with the minimal age

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class RegistrationServiceImplTest {
```
Comment (aaananas) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:13]: lets add test for password null

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+        registrationService.register(alice);
+        bob.setLogin("Alice");
+        assertThrows(RuntimeException.class, () -> {
+            registrationService.register(bob);
+        }, "The same login! Exception!");
+    }
+
+    @Test
+    void register_userName_Ok() {
+        User actual1 = registrationService.register(alice);
```
Comment (aaananas) [inline src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:74]:
```suggestion
        User aliceActual = registrationService.register(alice);
```
lets choose better names

**Review by aaananas — CHANGES_REQUESTED** [review]

File: src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java (hunk 3 above)
Comment (SofiiaKazanivska) [reply src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java:14]: Are you sure?

**Pull request closed without merging**